concepts/strings-package/about.md

```diff
@@ -1,5 +1,23 @@
 # About
 
-## Source code encoding
+The [`strings` package](https://pkg.go.dev/strings) contains many useful functions to work on strings:
 
-Source code files in Go are always `UTF-8` encoded. This makes it very easy to use special characters in strings.
+| Function                                            | Purpose                                                             |
+| --------------------------------------------------- | ------------------------------------------------------------------- |
+| [ToLower](https://pkg.go.dev/strings#ToLower)       | Convert the string to lower case                                    |
+| [ToUpper](https://pkg.go.dev/strings#ToUpper)       | Convert the string to upper case                                    |
+| [TrimSpace](https://pkg.go.dev/strings#TrimSpace)   | Trim leading and trailing spaces                                    |
```

Comment:
I think spaces means just "space characters", should say whitespace

Reply:
Totally